neuraxle/metaopt/auto_ml.py

```diff
@@ -23,10 +8,11 @@
 import traceback
 from abc import ABC, abstractmethod
 from enum import Enum
-from typing import List, Callable
+from typing import Callable, Dict, List, Union
 
-from neuraxle.base import MetaStepMixin, BaseStep, ExecutionContext, ForceHandleOnlyMixin, \
-    EvaluableStepMixin
+from sklearn.metrics import mean_squared_error
```

Comment:
Remove sklearn imports from this file. Sklearn imports should be minimal in case we may want to remove the dependency later on. 

Let's try to import scikit-learn only in a few examples or tests, and in the package `neuraxle.steps.sklearn` for instance. 

In the worst case: document that a possible function to use as a metric in the class AutoML is the `from sklearn.metrics import mean_squared_error` function.